packages/client/src/pages/example/example.tsx

```diff
@@ -167,12 +167,19 @@ const PostExample = () => {
   const vancouver: Location = { name: "Vancouver", lat: 49.26, lon: -123.22 };
   const surrey: Location = { name: "Surry", lat: 49.26, lon: -123.22 };
   const richmond: Location = { name: "Richmond", lat: 49.26, lon: -123.22 };
-  const yellowknife: Location = { name: "Yellowknife", lat: 49.26, lon: -123.22 };
+  const yellowknife: Location = {
+    name: "Yellowknife",
+    lat: 49.26,
+    lon: -123.22,
+  };
```

Comment:
did u change this formatting on purpose to emphasize that ur past wrongs are righted

Reply:
the good ppl of yellowknife deserve it